apk-generator/v2/app/generator/generator.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, config, via_api=False, identifier=None, task_handle=None):
         self.api_link = ''
         self.apk_path = ''
         self.via_api = via_api
+	self.build_type = build_type
```

Comment:
It'd be a command line argument http://how-to.wikia.com/wiki/How_to_read_command_line_arguments_in_a_bash_script